requirements.txt

```diff
@@ -27,3 +27,4 @@ vim-vint~=0.3.6
 # Use >=3.3 once nltk-3.3 is released.
 nltk~=3.1.0
 appdirs~=1.4.0
+
```

Comment:
rah, i don't know where did this creep in. Fixed. This is needed because @Makman2 did some changes which are required for tests to pass here. The `result_message` was added